assets/css/_properties_panel.scss

```diff
@@ -9,7 +9,7 @@
   right: 0;
   top: 0;
   width: 23.5rem;
-  z-index: 1002;
+  z-index: $right-panel-z-index;
```

Comment:
I looked into adding a new `right-panel` class as a single location to set the z-index property, but this did not resolve the z-index issue for the properties panel. 

Opting to create a single z-index constant instead set by each of the possible right-panel classes to resolve the immediate issue with a marginal improvement in consistency.

Reply:
Unfortunately, the `z-index` change does result in the header being visible again, but it changes the overall layout of the page versus before - now the right panel takes up 100% of the height of the browser and covers part of the top nav, as opposed to being positioned below the top nav. Interestingly, late view doesn't have this issue, and I'm not immediately sure why.

Reply:
Ah, I missed that subtlety and will re-visit! LateView is a separate component from RightPanel -- I'll dig a bit more into how it is positioned & replicate for the RightPanel components

Reply:
Bringing some consistency to our z-indices is still a really good addition, though!

Reply:
Thanks! Okay, I think the latest on dev-green reflects what is expected here: right panel is below the top bar, but in the case of VPP, above the modal overlay shadow. 

Doing this change & running into these visual inconsistencies makes me think about adding some sort of visual regression tests, especially since there are multiple breakpoints to check! I suspect that would be out of scope for this particular PR but can add a backlog ticket if it seems worthwhile. Looks like there [is already one](https://app.asana.com/0/1152340551558956/1202357245709641/f) for adding cypress for a different use case (accessibility testing)

Reply:
Yeah, agreed that that would be nice, especially given how different the various breakpoints are.